### PR TITLE
Change community.local_only to community.visibility

### PR DESCRIPTION
**Pull request by Nutomic**
This can also be used for private community in the future.

### CODE REVIEW

File: copy_generated_types_from_lemmy.sh [hunk 1]
```diff
@@ -24,13 +24,9 @@ cp utils/bindings/* ../../lemmy-js-client/src/types/
 
 popd
 
-# Remove the Sensitive type
-rm src/types/Sensitive.ts
-
```
Comment (Nutomic) [inline copy_generated_types_from_lemmy.sh:29]: Not being generated anymore.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "lemmy-js-client",
-  "version": "0.19.2-alpha.3",
+  "version": "0.19.2-alpha.4",
```
Comment (SleeplessOne1917) [inline package.json:3]: Should the version for this still be 0.19.2? I would expect 0.19.3-something.

**Comment by dessalines** [comment]
Make these draft PRs until the back-end is merged. If you need unit tests to pass on the back-end, then you can deploy an alpha RC off this branch.

File: package.json (hunk 2 above)
Comment (Nutomic) [reply package.json]: Fixed

**Comment by dessalines** [comment]
This setting's been merged.

**Pull request closed without merging**